### PR TITLE
Poetic minimalism as an ode to creators.

### PR DESCRIPTION
**Pull request by ntoll**
Please feel free to ignore this PR. It's merely evidence of my poetic and philosophical engagement with your project. Inspired by your README and after further thinking about #6 I felt creative and tried to rework it to as concentrated form as possible. In so doing, I realised I was (we are) in fact writing something poetic. At one point I even changed the title to "The Creator's [c]Ode". ;-)

In any case, I'll annotate my thinking to the diff so you can see how I found this a very stimulating 45 minutes of my time. If nothing else, this is a thanks in PR-form for such an opportunity.

As always, comments, constructive critique and ideas are most welcome!

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,21 +1,16 @@
-# Code of Conduct
+# The Creator's Code
```
Comment (ntoll) [inline CODE_OF_CONDUCT.md:1]: Or should that be "The Creator's [c]Ode". (I really don't like "Code of Conduct" as a term - it always makes me think of behaviour management policies at school.)

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
 
-We are sentient.  
-To be sentient is to be limited.  
-In our limitation, we make choices that are unwise or are flawed.
+Because we are limited beings,
+we make choices that are unwise or flawed.
```
Comment (ntoll) [inline CODE_OF_CONDUCT.md:4]: These two lines are the strophe - the set-up.

I really love the sentiment you express since it frames the whole of the rest of the text in humility and compassion.

* I've tried to keep the language simple and easy enough for young folks to be able to engage with (important for me since I spend lots of time working with young coders).
* Simple language is good because it's usually more accessible and easy to translate.
* Such sparse language also means there's plenty of space for the ideas to breath (it's not a heavy tome or legal contract).
* I've tried to avoid more philosophical terminology (sentient etc...) in favour of something philosophical but spoken clearly.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
 
-If we make unwise choices because of our limitation,  
-we cannot judge others for the same reason.
+While others may meet us with hostility,
+we choose to acknowledge difference:
```
Comment (ntoll) [inline CODE_OF_CONDUCT.md:7]: Here comes the antistrophe - the reply and balance to the opening.

In your original you give us a sort of categorical imperative (i.e. a universal rule) framed both in terms of ourselves and others (I really like that aspect).

However, upon reflection, I'm not sure about "judge" used in a negative sense. I "judge" things all the time -- for example, I've judged this as a worthwhile, interesting and thoughtful project..! ;-) Also, just because I make unwise choices, does that mean I cannot still make judgements..? In fact, I'd argue that judgement is an essential aspect of making choices... if you have good judgement, you'll make good choices, if you see what I mean. Only by reflecting on one's bad judgement will you move towards good judgement - so in a sense, bad judgement is an essential part of growing and enlarging one's world view. (Does this make sense..?)

So, I've tried to retain the feeling of a universal declaration with reference to ourselves and others. I've also framed it as a conscious choice (to acknowledge difference in the face of prejudice, intolerance and so on) which leads us to the most important line of the text...

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
 
-So, we cannot judge,  
-thus we forgive. 
+Thus we forgive.
```
Comment (ntoll) [inline CODE_OF_CONDUCT.md:9]: This is so important and makes a great conclusion to the antistrophe. It has impact, it's simply yet powerfully put and it captures the essence of this text (for me).

That's why I've put it on its own line in its own section (although it's still very much part of the antistrophe). :-)

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -1,21 +1,16 @@
...
-a place of learning,  
-a place of understanding,  
-a place of teaching,  
-a place of sharing,  
-a place of creators creating the tools for other creators to create complicated things elegantly.
+This is a place of learning, understanding, teaching and sharing.
+This is a place to affirm valour and compassion.
+This is a place for collaborative creation.
 
 Be well, Creator. Be well and create.
```
Comment (ntoll) [inline CODE_OF_CONDUCT.md:15]: I love this line.

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -1,21 +1,16 @@
...
 
-This project and its results are intended as:  
-a place of learning,  
-a place of understanding,  
-a place of teaching,  
-a place of sharing,  
-a place of creators creating the tools for other creators to create complicated things elegantly.
+This is a place of learning, understanding, teaching and sharing.
+This is a place to affirm valour and compassion.
+This is a place for collaborative creation.
```
Comment (ntoll) [inline CODE_OF_CONDUCT.md:13]: ...and so we come to the epode - the conclusion, completion or resolution.

* "This is a place" repeated for poetic/rhetorical effect.
* "a place of" (ingredients), "a place to" (actions), "a place for" (raison d'être).
* Shortening the sentences moves the pace forward and reaches for the climax (the final line).
* Obviously, I've taken your original list, massaged it as per my comment in #6 and added two nouns from your README that didn't make it into the original (but which I think complement and work with the other sentiments explained in this section).

Basically, I imagined this as the "recipe" for being a part of whatever endeavour this text is attached to.

**Review by ntoll — COMMENTED** [review]
Apologies for the linguistic analysis, but I was having lots of fun. ;-)

Like I said, don't hesitate to ignore any of this.

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (ntoll) [inline CODE_OF_CONDUCT.md:15]: I love this line... :-)

**Comment by ntoll** [comment]
Hi @Xe - closing this. ;-)

In the end I got to this blog post (published last August - just catching up on a long-term todo list item to close this PR): https://ntoll.org/article/on-being-together/

Thanks again for writing such an engaging and stimulating piece.

Peace.
